src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__img {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__list .nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  display: flex;
+  color: rgba(0, 0, 0, 1);
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 60px;
+  text-align: left;
+  align-items: center;
+  position: relative;
+}
+
+.is-active,
+.nav__link:hover {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: '';
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  bottom: 0;
+  display: block;
+  background-color: var(--main-color);
+  position: absolute;
+}
+
+.logo-link {
+  text-decoration: none;
+}
+
+a.logo-link img {
```

Comment:
The `display: block;` property here seems unnecessary. The `display: block;` property is typically used to make an element a block-level element. However, in this case, `a.logo-link img` is an `img` tag, which is already a block-level element by default. You can remove this property.